plugins/inputs/ceph/ceph.go

```diff
@@ -354,37 +354,65 @@ func (c *Ceph) execute(command string) (string, error) {
 
 // CephStatus is used to unmarshal "ceph -s" output
 type CephStatus struct {
+	FSMap struct {
+		NumIn        float64 `json:"in"`
+		NumMax       float64 `json:"max"`
+		NumUp        float64 `json:"up"`
+		NumUpStandby float64 `json:"up:standby"`
+	} `json:"fsmap"`
 	Health struct {
+		OverallStatus string `json:"overall_status"` // Only valid for ceph version <15
 		Status        string `json:"status"`
-		OverallStatus string `json:"overall_status"`
 	} `json:"health"`
+	MonMap struct {
+		NumMons float64 `json:"num_mons"`
+	} `json:"monmap"`
 	OSDMap struct {
-		OSDMap struct {
+		Epoch          float64  `json:"epoch"`
+		Full           bool     `json:"full"`     // Only valid for ceph version <15
+		NearFull       bool     `json:"nearfull"` // Only valid for ceph version <15
```

Comment:
I believe these won't exist in any version? <15 they will be in `osdmap` and >15 they don't exist.